shared/haml/hamburger.haml

```diff
@@ -0,0 +1,81 @@
+- extra_items_class = (defined? @level) ? "show-always" : "show-mobile"
+
+:ruby
+  teacher_entries = [
+    { title: "Home", url: "/teacher" },
+    { title: "Courses", url: "/educate" },
+    { title: "Projects Gallery", url: "/gallery" },
+    { title: "Sections", url: "/sections" },
+    { title: "Professional learning", url: "/pd" }
+  ].freeze
+
+  student_entries = [
+    { title: "Home", url: "/student" },
+    { title: "Courses", url: "/educate" },
+    { title: "Projects Gallery", url: "/gallery" }
+  ]
+
+  educate_entries = [
+    { title: "Educator Overview", url: "/educate" },
+    { title: "Elementary School", url: "/educate/curriculum/elementary-school'" },
+    { title: "Middle School", url: "/educate/curriculum/middle-school" },
+    { title: "High School", url: "/educate/curriculum/high-school" },
+    { title: "Hour of Code", url: "https://hourofcode.com" },
+    { title: "Partner with Us", url: "/educate/partner" },
+    { title: "Beyond Code.org", url: "/educate/curriculum/3rd-party" },
+    { title: "Inspire a Student", url: "/educate/resources/inspire" },
+    { title: "Online Community", url: "/educate/community" },
+    { title: "Tools and Videos", url: "/educate/resources/videos" },
+  ]
+
+  about_entries = [
+    { title: "About Us", url: "/about" },
+    { title: "Leadership", url: "/about/leadership" },
+    { title: "Donors", url: "/about/donors" },
+    { title: "Partners", url: "/about/partners" },
+    { title: "Full Team", url: "/about/team" },
+    { title: "Newsroom", url: "/about/news" },
+    { title: "Evaluation", url: "/about/evaluation" },
+    { title: "Jobs", url: "/about/jobs" },
+    { title: "Contact Us", url: "/contact" },
+    { title: "FAQs", url: "/faq" },
+  ]
```

Comment:
@Erin007:

- yes the new header has that bug link obscured.  This change does not implement red lines or neat layout.  Those will come next, and will be implemented before this new UI is shared.
- Without the special cookie, the legacy header and hamburger should appear.  We will test this thoroughly before merging.  (I guess it might have a few small changes, like the improved slideDown animation; I'll talk to you more about that when testing.)
- I'm not sure what you mean by "other side navs".